README.md

```diff
@@ -44,10 +44,17 @@ var flickr = new Flickr({
 });
 ```
 
+If you use `module` loader and you just use like this:
+```
+import Flickr from 'flickrapi/browser/flickrapi.dev.js'
+// do sth that use Flickr name
```

Comment:
please be as descriptive as possible here. Anyone should be able to understand a README, and "sth" is not going to be understood by a lot of people